astropy_helpers/tests/test_setup_helpers.py

```diff
@@ -135,7 +138,10 @@ def extension_test_package(tmpdir, request):
 
 @pytest.fixture
 def c_extension_test_package(tmpdir, request):
-    return _extension_test_package(tmpdir, request, extension_type='c')
+    # Check whether numpy is installed in the test environment
+    has_numpy = bool(importlib.find_loader('numpy'))
```

Comment:
This doesn't work on py2 on LTS, and also `Deprecated since version 3.4: Use importlib.util.find_spec() instead.`